### PR TITLE
Label.1

### PR DESCRIPTION
**Pull request by kem247**
Resolves issues #6 and #11

### CODE REVIEW

File: src/components/LabelBase/styles.module.css [hunk 1]
```diff
@@ -1,11 +1,19 @@
+:root {
+  font-size: 16px;
+}
```
Comment (mewtaylor) [inline src/components/LabelBase/styles.module.css:3]: I think we can take this out and I'll include it in my PR that migrates the header

File: src/components/LabelMenus/styles.module.css [hunk 2]
```diff
@@ -1,64 +1,71 @@
+:root {
+  font-size: 16px;
+}
```
Comment (mewtaylor) [inline src/components/LabelMenus/styles.module.css:3]: Can take this out

File: src/components/LabelMenus/index.js [hunk 3]
```diff
@@ -20,47 +22,58 @@ class LabelMenus extends Component {
...
-  return {
-    title: state.title,
-  }
-}
-export default connect(mapState)(LabelMenus)
+// const mapState = state => {
+//   return {
+//     title: state.title,
+//   }
+// }
```
Comment (mewtaylor) [inline src/components/LabelMenus/index.js:78]: Remove commented code

File: src/components/LabelMenus/styles.module.css [hunk 4]
```diff
@@ -1,64 +1,71 @@
...
-  border: solid 1px #707070;
+  height: 13rem;
+  border: solid 0.0625rem #707070;
   background-color: #f4f4f4;
-  margin: 35px 40px;
-  padding: 30px 0px 0px 0px;
-  position: relative;
+  margin: 2.1875rem 0rem;
+  padding: 1.875rem 0rem 0rem 0rem;
+  /* position: relative; */
```
Comment (mewtaylor) [inline src/components/LabelMenus/styles.module.css:15]: remove commented code

File: src/components/LabelMenus/styles.module.css [hunk 5]
```diff
@@ -1,64 +1,71 @@
...
 }
 .highlightLabel {
   background-image: url("/highlightbar.png");
 
-  background-size: 8px 560px;
+  background-size: 0.5rem 35rem;
   background-repeat: no-repeat;
 }
 
 .lorem {
```
Comment (mewtaylor) [inline src/components/LabelMenus/styles.module.css:24]: rename to `.labelMenuDescription` or something like that to describe what will eventually go in this section

File: src/components/LabelMenus/styles.module.css [hunk 6]
```diff
@@ -1,64 +1,71 @@
...
   flex: 1;
   display: flex;
-  width: 82px;
-  height: 14px;
+  width: 5.125rem;
+  height: 0.875rem;
   font-family: Helvetica;
```
Comment (mewtaylor) [inline src/components/LabelMenus/styles.module.css:29]: Let's keep fonts uniform for now, and remove tihs

File: src/components/LabelBase/styles.module.css [hunk 7]
```diff
@@ -1,11 +1,19 @@
...
 .labelBase {
+  width: 75%;
+  height: 41.9375rem;
+  border: solid 0.0625rem #707070;
+  background-color: #f4f4f4;
+  margin-top: 2.1875rem;
+  /* /* margin-left: 370px; */
```
Comment (mewtaylor) [inline src/components/LabelBase/styles.module.css:10]: remove commented code

File: src/components/_tests_/LabelMenus.test.js [hunk 8]
```diff
@@ -1,10 +1,41 @@
...
+    wrapper.find("#selectButton").forEach(node => {
+      node.simulate("click")
+    })
+    expect(selectItem).toHaveBeenCalledTimes(3)
+
+    // const title = wrapper.find({ state: props.highlightValue })
+    // expect(title.to.have.lengthOf(1))
+    // expect(wrapper.state().showHighlight).toBe(false)
+    // title.simulate("click")
+    // expect(wrapper.state().showHighlight).toBe(true)
```
Comment (mewtaylor) [inline src/components/_tests_/LabelMenus.test.js:39]: remove commented code

File: src/components/LabelMenus/styles.module.css (hunk 6 above)
Comment (kem247) [reply src/components/LabelMenus/styles.module.css:29]: Hi,  on my end i don't see anything with pixels in that file

**Pull request merged**